src/App.css

```diff
@@ -26,3 +26,86 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+
+.container{
+  background: white;
+  width: 400px;
+  height: 247px;
+  padding-left: 15px;
+  margin-top: 20px;
+  display: inline-block;
+  margin-right: 20px;
+}
+
+.prueba{
```

Comment:
English please, If you are not using this 'class' please remove it.

Reply:
Done.